src/api/postItemToServer.js

```diff
@@ -0,0 +1,23 @@
+//Helpers
+import _uniqueId from 'lodash/uniqueId';
+import { BASE_URL } from '../constants/baseUrl';
+
+function getPostInitData(postData) {
+  return {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify(postData),
+  };
+}
+
+export default function postItemToServer(key, value) {
+  const postData = {
```

Comment:
this data should be directly coming from wherever this function has been called, I think you should add the Id their itself